nuxeo-mongodb-core/src/test/java/org/nuxeo/mongodb/core/MongoDBComponentFeature.java

```diff
@@ -74,15 +73,16 @@ public void afterTeardown(FeaturesRunner runner) {
         MongoDBComponent mongoComponent = (MongoDBComponent) Framework.getRuntime().getComponent(
                 "org.nuxeo.mongodb.core.MongoDBComponent");
         // Get database
-        MongoDatabase database = mongoComponent.getDatabase();
-        for (String collName : database.listCollections().map(doc -> doc.get("name").toString())) {
-            // Filter out collections used by repositories (eg: 'default' and 'default.counters')
-            if (repositoryNames.stream().anyMatch(collName::startsWith)) {
-                continue;
+        mongoComponent.getDatabases().forEach(database -> {
+            for (String collName : database.listCollections().map(doc -> doc.get("name").toString())) {
```

Comment:
I'm not sure having a `map` inside a `for` loop is useful or clearer, you put the code directly in the loop.